lib/tasks/feature_flags.rake

```diff
@@ -0,0 +1,12 @@
+FEATURES_YML_PATH = 'config/feature-flags.yml'.freeze
```

Comment:
`/Users/user/dev/rails_api_base/config/initializers/flipper.rb:1: warning: already initialized constant FEATURES_YML_PATH`

dropping constants like this is a problem, they get defined in the global namespace and are even conflicting between themselves. I'd just use the string directly in this case

Reply:
good catch